components/tooltip/tooltip.js

```diff
@@ -582,18 +582,17 @@ class Tooltip extends RtlMixin(LitElement) {
 		this._openDir = space.dir;
 
 		// Compute the x and y position of the tooltip relative to its target
-		let parentTop;
-		let parentLeft;
+		let offsetTop, offsetLeft;
```

Comment:
`parent` was just confusing me here, but if it makes more sense to others I can revert this.